app/components/sdg_management/homepage/show_component.rb

```diff
@@ -1,14 +1,19 @@
 class SDGManagement::Homepage::ShowComponent < ApplicationComponent
   include Header
 
-  attr_reader :phases
+  attr_reader :phases, :header_card
 
-  def initialize(phases)
+  def initialize(phases, header_card)
```

Comment:
Since the `header` is listed before the `phases`, the order of the parameters is a bit unintuitive :thinking:.

Would named parameters make sense here again? In general, I prefer named parameters when it's impossible for me to remember their order :sweat_smile:.